curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64c758ab7352130b775df8c4.md

```diff
@@ -11,10 +11,10 @@ In the map callback function, pass in `platform` for the parameter and implicitl
 
 # --hints--
 
-Your map callback should implicitly return the creation of a new `Platform` instance with the `platform.x` and `platform.y` values passed in as arguments.
+Your `map` callback should implicitly return the creation of a new `Platform` instance with the `platform.x` and `platform.y` values passed in as arguments.
 
 ```js
-assert.match(code, /const\s+platforms\s*=\s*platformPositions\.map\(\s*\(\s*platform\s*\)\s*=>\s*new\s+Platform\s*\(\s*platform\.x,\s*platform\.y\s*\)\s*\);\s*/
+assert.match(code, /const\s+platforms\s*=\s*platformPositions\.map\(\s*\(?\s*platform\s*\)?\s*=>\s*new\s+Platform\s*\(\s*platform\.x,\s*platform\.y\s*\)\s*\);\s*/
```

Comment:
```suggestion
assert.match(code, /const\s+platforms\s*=\s*platformPositions\.map\(\s*(\(\s*platform\s*\)|platform)\s*=>\s*new\s+Platform\s*\(\s*platform\.x,\s*platform\.y\s*\)\s*\);\s*/
```
This will match only with both parenthesis, or with none.

Reply:
@gikf and @JamesCartar can you put a `?` after the semicolon? so that it becomes optional for the users to put it there while solving ?